lib/objectid.js

```diff
@@ -0,0 +1,5 @@
+var ObjectId = require('mongodb').ObjectId;
```

Comment:
Another option is to optionally depend on `mongodb` (or similar, eg. an `ObjectID` module). Sort of like this:

```js
```js
try {
  mongodb = require('mongodb');
} catch(e) {
  if (!isNotFound(err)) { 
    throw e;
  }
}
```